src/main/java/com/autotune/database/dao/ExperimentDAOImpl.java

```diff
@@ -75,6 +78,43 @@ public ValidationOutputData addExperimentToDB(KruizeExperimentEntry kruizeExperi
         return validationOutputData;
     }
 
+    @Override
```

Comment:
Please add a javadoc here explaining what is `daysCount` referring to.
Also I see most of the methods in this class are missing a javadoc, that makes it hard to review the code in general. Would appreciate if you can add a few lines for each of them

Reply:
Yes Its already mentioned [here](https://github.com/kruize/autotune/blob/5ba4300e8f9d76a6a2b846a47613156a5d41720e/design/KruizeConfiguration.md#other-configuration) 
but will add comments

Reply:
done